lib/smart_answer/calculators/minimum_wage_calculator.rb

```diff
@@ -108,7 +108,11 @@ def historical_adjustment
     end
 
     def minimum_wage_or_above?
-      minimum_hourly_rate <= total_hourly_rate
+      minimum_entitlement <= total_pay
```

Comment:
```suggestion
      historical_entitlement <= total_pay
```

It doesn't feel like we need the `minimum_entitlement` method since it just returns the value of a different method.

Reply:
I had this discussion with myself at length, I only added that cause `historical_entitlement` is used only in the context of checking historical wages. I just felt `minimum_entitlement` was clearer when reading the code, I'm happy to be proven wrong though.

Reply:
That's reasonable. Rather than create a new method, you can use [`alias`](https://apidock.com/ruby/Module/alias_method) which is slightly nicer!

```rb
alias_method :historical_entitlement, :minimum_entitlement
```

Reply:
Ooh I had absolutely no idea this existed! Thanks Tom!

Reply:
For some reason `alias_method` wouldn't work in this instance, seemed to have a scoping issue as that call eventually led to a private method which was erroring.